app/templates/contact/index.html

```diff
@@ -15,7 +15,7 @@ <h1 class="ui header">
         {% endif %}
         {{ page.render_inline_editor(editable_html_obj, current_user) }}
         {% if not current_user.is_admin() %}
-            <div class="ui divider"></div>
+            <!--<div class="ui divider"></div>--> 
```

Comment:
we want to keep this divider since we'll have text above the contact form and we need to make it seem separate